src/components/widget-player-page.jsx

```diff
@@ -92,14 +92,13 @@ const WidgetPlayerPage = () => {
 	if ( type == EMBED || type == PREVIEW_EMBED || type == LEGACY_EMBED ) headerRender = null
 
 	let bodyRender = null
-	if( (!!state.widgetID) && state.playID !== undefined ) {
```

Comment:
I'm a little nervous about rendering `bodyRender` unconditionally, just because the widget player is always expecting the `instanceId` and `playId` props to exist

Reply:
It's good to leave it in 👍🏻